dependencies/maven/artifacts.bzl

```diff
@@ -23,7 +23,6 @@ artifacts = [
     "com.google.code.findbugs:jsr305",
     "com.google.errorprone:error_prone_annotations",
     "com.google.guava:guava",
-    "com.google.guava:listenablefuture",
```

Comment:
The fix to explicitly include listenable future for windows building was not necesary, it turns out!